solvexia_sdk/file/file.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
+
+import requests
+import json
+import sys
+import math
+import os
+from fsplit.filesplit import Filesplit
+from solvexia_sdk import api
+
+class file:
+    def __init__(self, fileId):
+        self.fileId = fileId
+
+    def getFileMetadata(self):
+        response = api.apiGet(f"files/{self.fileId}/metadata")
+        api.statusCodeCheck(response, "Error getting metadata for file")
+        return response.json()
+
+    def updateMetadata(self, name):
+        payload = {
+            'name': name
+        }
+        response = api.apiPost(f"files/{self.fileId}/metadata", payload)
+        api.statusCodeCheck(response, "Error updating file metadata")
+        return response.json()
+
+    def uploadFile(self, file):
+        uploadFileUrl = api.baseUrl + f"files/{self.fileId}"
+        with open(file, "rb") as openFile:
+            response = requests.post(uploadFileUrl, files={"Filename": openFile}, headers=api.access_token)
+        api.statusCodeCheck(response, "Error uploading file")
+        print(response.json())
```

Comment:
Is there any reason to print response json here?

Reply:
forgot to remove after finishing debugging and testing